internal/pkg/service/stream/storage/level/local/encoding/encoder/csv/csv_benchmark_test.go

```diff
@@ -200,15 +206,9 @@ func BenchmarkCSVWrite(b *testing.B) {
 
 func newBenchmark(configure func(wb *benchmark.WriterBenchmark)) *benchmark.WriterBenchmark {
 	columns := column.Columns{
-		// 8 columns, only the count is important for CSV
 		column.UUID{},
 		column.Datetime{},
 		column.Body{},
-		column.Template{},
-		column.Template{},
-		column.Template{},
-		column.Template{},
-		column.Template{},
 	}
```

Comment:
The benchmark now includes the `column.Renderer` so it must be updated.

Speeds are same, ... but there is no more zero memory allocations, because, the renderer must allocate some memory (6 allocs per op).